crates/bevy_core_pipeline/src/taa/mod.rs

```diff
@@ -226,9 +236,7 @@ impl ViewNode for TAANode {
                         },
                         BindGroupEntry {
                             binding: 3,
-                            resource: BindingResource::TextureView(
-                                &prepass_depth_texture.default_view,
-                            ),
+                            resource: BindingResource::TextureView(&depth_view),
```

Comment:
How cheap is it to create textureviews? I've been wondering if it's worth it to change the default_view that we create+store in the cache to something user controlled, as iirc we don't end up using the default_view on a lot of the cached textures, but we still create it and store it.

Not something that needs to be fixed, just a possible future improvement to bevy's renderer.

Reply:
That's a good question. I asked cwfitzgerald about making them every frame in some passes and they said:
> shouldn't be [expensive]
like dont make a thousand if you can help it
but a couple dozen, meh